### PR TITLE
Add skipGodeps flag to repo rules

### PR DESCRIPTION
**Pull request by sttts**
TODO:

- [ ] test

### CODE REVIEW

File: artifacts/scripts/util.sh [hunk 1]
```diff
@@ -596,6 +596,10 @@ function git-index-clean() {
 }
 
 function fix-godeps() {
+    if [ "${SKIP_GODEPS:-}" = true ]; then
```
Comment (mfojtik) [inline artifacts/scripts/util.sh]: can we make this more scoped to publisher bot? like: `PUBLISHER_BOT_SKIP_GODEPS`

Comment (sttts) [reply artifacts/scripts/util.sh]: done

**Pull request merged**